addons/mail/controllers/thread.py

```diff
@@ -10,7 +10,7 @@
 
 
 class ThreadController(http.Controller):
-    @http.route("/mail/thread/data", methods=["POST"], type="json", auth="user")
+    @http.route("/mail/thread/data", methods=["POST"], type="json", auth="public")
```

Comment:
Why this change? Is there any chance that a public user would have access to the thread data without token and without sudo?

I feel like maybe we'd want to duplicate the routes for portal, similarly to what we did for livechat cors?

And then those routes only can be flagged with public and protected with the check?

Reply:
I don't think that access without token or sudo is possible, but if the `auth=user`, it's not possible to access this controller for non users at all.